src/providers/store-provider/storeContext.ts

```diff
@@ -0,0 +1,16 @@
+import { createContext } from 'react';
+
+export interface IStoreContext {
+  selectedCity: string;
+}
+
+export type StoreDispatchType = (storeProperty: Partial<IStoreContext>) => void;
+
+export const defaultStore: IStoreContext = {
+  selectedCity: '',
+};
+
+export const defaultStoreDispatch: StoreDispatchType = () => {};
```

Comment:
clarify pls a bit what is defaultStoreDispatch

Reply:
It's just a default value for dispatch function stored in DispatchContext.
Added here empty function as a fallback to avoid using `null` or smth else.